### PR TITLE
Feature/ng9 ivy integration tests

### PR DESCRIPTION
**Pull request by poloagustin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

Rename `integrations/ivy` to `integrations/hello-world-ng9-ivy` to be consistent with the other integration tests.
Create a non-ivy version of the hello world integration tests project under `integrations/hello-world-ng9-ivy-off`

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
New integration tests for angular 9 without ivy

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by splincode** [comment]
`yarn integration:ng9-ivy-off`

image

**Comment by poloagustin** [comment]
Not sure how to fix the travis bug. Is it something related to ivy?

**Comment by splincode** [comment]
Circle CI run isolated every step, but Travis CI running only once in own evironments

try running every each step by step (.travis-ci.yml) in your local machine

File: package.json [hunk 1]
```diff
@@ -106,7 +106,8 @@
     "// - INTEGRATION": "Integration builds",
     "integration:ng6": "cd integrations/hello-world-ng6 && yarn test",
     "integration:ng8": "cd integrations/hello-world-ng8 && yarn test",
-    "integration:ivy": "cd integrations/ivy && yarn test:ivy"
+    "integration:ng9-ivy": "cd integrations/hello-world-ng9-ivy && yarn test:ivy",
```
Comment (joaqcid) [inline package.json:109]: maybe `"integration:ng9:ivy"`

Comment (markwhitfeld) [reply package.json:109]: agreed, would be best to keep it consistent

**Comment by splincode** [comment]
any progress?

**Comment by markwhitfeld** [comment]
Hi @poloagustin 
Have you been able to work on this.

**Comment by poloagustin** [comment]
build completes now, @markwhitfeld !

**Review by markwhitfeld — APPROVED** [review]
Awesome!!!

**Comment by markwhitfeld** [comment]
Merging this because it was just the build blocking other approvals.

**Pull request merged**